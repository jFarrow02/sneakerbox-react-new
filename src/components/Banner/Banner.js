import React from 'react';
import './Banner.scss';
import {
    Cart,
} from '../index';

/**
 * 
 * This component renders a header component to be displayed at the top of each application page. It also contains
 * a Cart component to display the user's shooping cart.
 */
const Banner = props=>{

    return(
        <header className='Banner'>
            <h1>{props.title}</h1>
                <Cart/>
        </header>
    )
}

export default Banner;