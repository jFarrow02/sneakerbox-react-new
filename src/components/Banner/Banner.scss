$darkblue : darkblue;

.Banner{
    border: 1px solid $darkblue;
}