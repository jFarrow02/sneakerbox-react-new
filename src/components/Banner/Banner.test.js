import React from 'react';
import renderer from 'react-test-renderer';
import Banner from './Banner';

it('matches a snapshot', ()=>{

    const tree = renderer.create(<Banner/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})