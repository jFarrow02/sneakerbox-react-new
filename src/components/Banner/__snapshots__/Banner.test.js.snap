// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches a snapshot 1`] = `
<header
  className="Banner"
>
  <h1 />
  <div
    className="Cart"
  >
    Cart goes here
    <h4>
      No Items
    </h4>
  </div>
</header>
`;
